### ISSUE TITLE
Gii: no such cmd: listCollections

### ISSUE THREAD

**Issue by purocean** [yiisoft/yii2-mongodb#188]
### What steps will reproduce the problem?
visit http://localhost:8080/gii/mongoDbModel
### What do you get instead?
```
MongoDB Exception – yii\mongodb\Exception
no such cmd: listCollections
Caused by: MongoDB\Driver\Exception\RuntimeException
no such cmd: listCollections

in xxx\vendor\yiisoft\yii2-mongodb\Command.php at line 187
```

### Additional info

| Q                      | A
| ---------------------- | ---
| Yii version            | 2.0.11.2
| Yii MongoDB version    | 2.1.3.0
| MongoDB server version | 3.2.12
| PHP version            | 7.0.6
| Operating system       | Windows 10 x64

**Comment by klimov-paul** [comment]
Unable to reproduce: I have just generated new MongoDB ActiveRecord with latest code.
Command `listCollections` is available at MongoDB since 3.0 :
https://docs.mongodb.com/manual/reference/command/listCollections/

**Comment by purocean** [comment]
resolved! I just restart mongodb service...
